addons/website_slides/static/src/js/slides_embed.js

```diff
@@ -227,22 +227,32 @@ $(function () {
             var widget = $('.oe_slide_js_share_email');
             var input = widget.find('input');
             var slideID = widget.find('button').data('slide-id');
-            if (input.val() && input[0].checkValidity()) {
```

Comment:
The runbot is a bit old so I can't test it, but wasn't it done to check if the input email address was correctly formatted?

Reply:
Hello @awa-odoo 

Yes, it was done to validate email address, but now we are supporting multiple email addresses hence it is changed.
we can use Email Input with `multiple="True"` attribute. But, The custom 'email' keyboard does not provide a comma key, so users cannot enter multiple email addresses.